PWGHF/D2H/Tasks/CMakeLists.txt

```diff
@@ -74,6 +74,11 @@ o2physics_add_dpl_workflow(task-flow-charm-hadrons
                     PUBLIC_LINK_LIBRARIES O2Physics::AnalysisCore O2Physics::EventFilteringUtils
                     COMPONENT_NAME Analysis)
 
+o2physics_add_dpl_workflow(task-directed-flow-charm-hadrons
+                    SOURCES taskDirectedFlowCharmHadrons.cxx
+                    PUBLIC_LINK_LIBRARIES O2Physics::AnalysisCore O2Physics::EventFilteringUtils
+                    COMPONENT_NAME Analysis)
```

Comment:
Respect alphabetic order.
Where do you use `EventFilteringUtils`?

Reply:
[Reply]: I updated the CMakeLists.txt....Is it ok now?
              I used PWGHF/Utils/utilsEvSelHf.h which i guess needs EventFilteringUtils. It is same as used in https://github.com/AliceO2Group/O2Physics/blob/master/PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx.